.github/workflows/pipeline.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   linting:
     name: "Running ${{ matrix.linter }}"
     runs-on: ubuntu-latest
-    container: ruby:2.5
+    container: ruby:2.7
```

Comment:
This change was required in order to stop bundler complaining with the decidedly unhelpful message "Your bundle is locked to rails (7.0.1) from rubygems repository [rubygems.org](http://rubygems.org/) or installed locally, but that version can no longer be found in that source. That means the author of rails (7.0.1) has removed it."